### ISSUE TITLE
Safing journey and Android Support

### ISSUE THREAD

**Issue by NolonQ** [safing/portmaster#1655]
**What would you like to add or change?**:
I don't have discord so i'm asking here instead.
When the pc security channel covered portmaster, a wave of newcomers purchased the spn, this peak seemingly dropped.
What's the current status of safing being funded purely by "community donations"?
What's the progress of portmaster for android? Portmaster for android would be just so amazing.

**Why do you and others need this?**:

**Comment by NolonQ** [comment]
Kein kommentar?

**Comment by Victor239** [comment]
Still relevant.

**Comment by dhaavi** [comment]
Hey, sorry, this got lost in the weeds.

> When the pc security channel covered portmaster, a wave of newcomers purchased the spn, this peak seemingly dropped.

Yes, this was expected to be a peak.

> What's the current status of safing being funded purely by "community donations"?

Our goal never was to be funded by community donations.
There are donations, but they are there because people asked about it and we wanted to give another option to contribute.
Safing is fully funded by subscribers.

> What's the progress of portmaster for android? Portmaster for android would be just so amazing.

There is an early beta version that includes just the SPN.
We still need more resources to invest resources into Android again.

---

Some more details:

We started sharing the growth progress in the news widget, because we saw people reaching out and wanting to see Safing and Portmaster succeed.
Our intention was to get users more involved in the progress and growth of Portmaster, but it turned out that this was not a suitable way to do so.
When things got more complicated to explain and display - without causing confusion - we decided to remove it again.

Bottom line: Safing is still on track and growth is steady.

The best place to get regular updates on Safing and Portmaster, is to (re)watch our Safing Live every other week: https://www.youtube.com/c/safingio

**Comment by Victor239** [comment]
@dhaavi I'm not interested in reading every update, I just want to follow when the Android app is available as it's the main blocker for me using SPN. Is there a GitHub issue or mailing list I can follow for **just** the Android app progress?

**Comment by dhaavi** [comment]
You can subscribe to the newsletter at the bottom of our homepage: https://safing.io/
